update_deps/update_deps.go

```diff
@@ -0,0 +1,103 @@
+package main
```

Comment:
Buildbarn repositories use `rules_go` to build Go binaries. Could you please add Bazel build infrastructure to this repository to be able to build and run this tool? I seem to remember @mickael-carl is also planning on adding some bits to this repository that need this. @mickael-carl, could you please work together with @zoidbergwill to ensure there is no race condition?

Reply:
Totally can do. Wasn't even sure this script was something you folks would be interested in maintaining. 😁